bio.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+	<title>Bio | Chandler Morrison</title>
+	<link href="https://fonts.googleapis.com/css?family=Inconsolata|Montserrat|Oxygen" rel="stylesheet">
+<ul class="NavBar">
+	<li><a href="index.html">Home</a></li>
+	<li><a href="bio.html">Bio</a></li>
+</ul>
+
+</head>
+<body background="images/Jacket.jpg">
+<h1>About Me</h1>
+<img src="images/Me.jpg">
+
+
+
+
+
```

Comment:
Why all the empty space? This could lead to increased difficulty for teammates to read through code. Always aim to be cohesive and consistent in your spacing and indentation.